### PR TITLE
Fill NA values in causal impact input

### PR DESCRIPTION
**Pull request by hideaki**
### Description
Fill NA values in causal impact input

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/causal_impact.R [hunk 1]
```diff
@@ -34,6 +34,13 @@ do_causal_impact <- function(df, time, formula, ...) {
 #' @param time_col - Column that has time data
 #' @param formula - Formula with target value column on the left-hand side, and predictor columns on the right-hand side. e.g. y ~ predictor1 + predictor2
 #' @param intervention_time - The point of time where intervention happened.
+#' @param na_fill_type - Type of NA fill:
```
Comment (double-y) [inline R/causal_impact.R]: How about making this argument na_fill and if it is numeric, regarding that it's the value to fill NA?

Comment (hideaki) [inline R/causal_impact.R]: Thought about that but did not do that since it would be more complex to generate the command from UI definition JSON.

Comment (double-y) [inline R/causal_impact.R]: ok

File: R/causal_impact.R [hunk 2]
```diff
@@ -95,6 +96,22 @@ do_causal_impact_ <- function(df, time_col, formula, intervention_time = NULL, o
...
+    }
+    else if (na_fill_type == "locf") {
+      df_zoo <- zoo::na.locf(df_zoo)
+    }
+    else if (na_fill_type == "StructTS") {
+      df_zoo <- zoo::na.StructTS(df_zoo)
+    }
+    else if (na_fill_type == "value") {
+      df_zoo <- zoo::na.fill(na_fill_value)
+    }
```
Comment (double-y) [inline R/causal_impact.R]: No need to add else clause?
Maybe NA values should be filtered?

Comment (double-y) [inline R/causal_impact.R]: Or maybe it should throw an error

Comment (hideaki) [inline R/causal_impact.R]: Added logic to throw error.

**Pull request merged**